go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/astronomer/astro-cli
 
 go 1.23.0
 
-// toolchain go1.24.1
+toolchain go1.24.1
```

Comment:
Do we need this toolchain entry? Should we just update the go version to 1.24?

Reply:
good question.
I have issues when this line is present when using debugger in IDE.
I have had issues in the past when this line is missing when running `make fmt` (gofumpt under the hood).

Reply:
I don't consider it blocking but it would be nice for the values to agree I guess? Anyway, approving as it is commented in main.